tests/unit/s2n_self_talk_custom_io_test.c

```diff
@@ -145,6 +146,8 @@ int main(int argc, char **argv)
     EXPECT_SUCCESS(s2n_config_add_cert_chain_and_key(config, cert_chain_pem, private_key_pem));
     EXPECT_SUCCESS(s2n_config_add_dhparams(config, dhparams_pem));
 
+    signal(SIGPIPE, SIG_IGN);
```

Comment:
Can you add a comment here on why we need this?

Reply:
Updated via amend